packages/react/src/components/SimplePagination/SimplePagination.jsx

```diff
@@ -48,56 +60,63 @@ const SimplePagination = ({
   const handlePrev = () => onPage(page - 1);
 
   return (
-    <div className={`${iotPrefix}-simple-pagination-container`}>
-      <span className={`${iotPrefix}-simple-pagination-page-label`} maxpage={maxPage}>
-        {pageText ? `${pageText} ${page}` : pageOfPagesText(page, maxPage)}
-      </span>
-      {maxPage > 1 ? (
-        <>
-          <div
-            className={
-              hasPrev
-                ? `bx--pagination__button bx--pagination__button--backward ${iotPrefix}-addons-simple-pagination-button`
-                : `bx--pagination__button bx--pagination__button--backward ${iotPrefix}-addons-simple-pagination-button-disabled`
-            }
-            role="button"
-            tabIndex={hasPrev ? 0 : -1}
-            onClick={hasPrev ? handlePrev : undefined}
-            onKeyDown={hasPrev ? (evt) => handleEnterKeyDown(evt, handlePrev) : undefined}
-          >
-            <CaretLeft16
-              dir="ltr"
-              description={prevPageText}
+    <div className={`${iotPrefix}-simple-pagination-container`} data-testid={testID}>
+      {totalItems ? (
+        <span className={`${iotPrefix}-simple-pagination-page-label`} maxpage={maxPage}>
+          {`${totalItems} ${totalItemsText}`}
+        </span>
+      ) : null}
+      <div className={`${iotPrefix}-simple-pagination-page-bar`}>
+        <span className={`${iotPrefix}-simple-pagination-page-label`} maxpage={maxPage}>
+          {pageText ? `${pageText} ${page}` : pageOfPagesText(page, maxPage)}
+        </span>
+        {maxPage > 1 ? (
+          <>
+            <div
               className={
                 hasPrev
-                  ? `${iotPrefix}-simple-pagination-caret`
-                  : `${iotPrefix}-simple-pagination-caret-disabled`
+                  ? `bx--pagination__button bx--pagination__button--backward ${iotPrefix}-addons-simple-pagination-button`
+                  : `bx--pagination__button bx--pagination__button--backward ${iotPrefix}-addons-simple-pagination-button-disabled`
```

Comment:
We should also try to follow BEM by adding `--` between the prefix and component name, and adding `__` between the main component and subcomponent names. For specific behaviors or differences in the subcomponent, add `--`
```suggestion
                  `${prefix}---pagination__button ${prefix}--pagination__button--backward ${iotPrefix}--simple-pagination__button--disabled`
```

Reply:
It looks like the component was previously written like this , but it wouldn't hurt to update to the proper standard